10320.xml

```diff
@@ -0,0 +1,153 @@
+CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+-->
+
+<LWM2M xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://openmobilealliance.org/tech/profiles/LWM2M.xsd">
+  <Object ObjectType="MODefinition">
+    <Name>CCU</Name>
+    <Description1><![CDATA[This LWM2M Object includes CCU(Central Control Unit) information and the link to CCU APPs.]]></Description1>
+    <ObjectID>10320</ObjectID>
+    <ObjectURN>urn:oma:lwm2m:x:10320</ObjectURN>
+    <LWM2MVersion>1.0</LWM2MVersion>
+    <ObjectVersion>1.0</ObjectVersion>
+    <MultipleInstances>Multiple</MultipleInstances>
+    <Mandatory>Optional</Mandatory>
+    <Resources>
+	  <Item ID="1">
+        <Name>CCU ID</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>String</Type>
+        <RangeEnumeration/>
+        <Units/>
+        <Description><![CDATA[The CCU identity.]]></Description>
+      </Item>
+      <Item ID="2">
+        <Name>CCU FM Version</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>String</Type>
+        <RangeEnumeration/>
+        <Units/>
+        <Description><![CDATA[The firmware version of the CCU.]]></Description>
+      </Item>
+      <Item ID="3">
+        <Name>CCU SW Version</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>String</Type>
+        <RangeEnumeration/>
+        <Units/>
+        <Description><![CDATA[The software version of the CCU.]]></Description>
+      </Item>
+	  <Item ID="4">
+        <Name>CCU Memory Size</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration/>
+        <Units>G</Units>
+        <Description><![CDATA[Total Memory Size of the CCU (expressed in gigabyte).]]></Description>
+      </Item>	  
+	  <Item ID="5">
+        <Name>CCU Storage</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration/>
+        <Units>M</Units>
+        <Description><![CDATA[Total storage of the CCU.]]></Description>
+      </Item>
+	  <Item ID="6">
+        <Name>CCU Available Storage</Name>
+        <Operations>R</Operations>
+        <MultipleInstances>Single</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>Integer</Type>
+        <RangeEnumeration/>
+        <Units>M</Units>
```

Comment:
Incorrect unit applied
If M is for meter, SenML defines this as ``m``

Reply:
Changed from M to GB.

Reply:
GB is the storage unit, Gigabyte.